ckanext/scheming/presets.json

```diff
@@ -144,6 +144,14 @@
         "form_snippet": "markdown.html",
         "display_snippet": "markdown.html"
       }
+    },
+    {
+      "preset_name": "radio",
+      "values": {
+        "form_snippet": "radio.html",
+        "display_snippet": "radio.html",
```

Comment:
why not use the existing "select.html" display snippet since both are displayed the same